internal/docstore/query.go

```diff
@@ -112,58 +156,65 @@ func (q *Query) BeforeQuery(f func(asFunc func(interface{}) bool) error) *Query
 func (q *Query) Get(ctx context.Context, fps ...FieldPath) *DocumentIterator {
 	dcoll := q.coll.driver
 	wrapErr := func(err error) error { return wrapError(dcoll, err) }
-	if err := q.init(fps); err != nil {
+	if err := q.initGet(fps); err != nil {
 		return &DocumentIterator{err: wrapErr(err)}
 	}
 	it, err := dcoll.RunGetQuery(ctx, q.dq)
 	return &DocumentIterator{iter: it, wrapError: wrapErr, err: wrapErr(err)}
 }
 
+func (q *Query) initGet(fps []FieldPath) error {
```

Comment:
Should this also check whether `OrderByField` also appears in the filters?

Reply:
Done.

I also added implementations for mem, mongo and firestore.

PTAL.

Reply:
Forgot to push?

Reply:
Right :(

PTAL